Bottom: 774b4e7c039a9a06577acf93861d5a87e9ccf6d8
Top:    b41f5d11d3f72c1bbe451336c858f7ebfb0a4da7
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-30 17:09:17 -0700

Refresh of addrinfo-af-ib

---

diff --git a/src/addrinfo.c b/src/addrinfo.c
index a89aa88..9c43043 100755
--- a/src/addrinfo.c
+++ b/src/addrinfo.c
@@ -171,7 +171,7 @@ static int ucma_convert_to_rai(struct rdma_addrinfo *rai,
 	}
 
 	if (ai->ai_flags & AI_PASSIVE) {
-		rai->ai_flags = RAI_PASSIVE
+		rai->ai_flags = RAI_PASSIVE;
 		if (ai->ai_canonname)
 			rai->ai_src_canonname = strdup(ai->ai_canonname);
